documentation/src/main/markdown/components/detectors.dita

```diff
@@ -824,6 +824,38 @@
 							</entry>
 							<entry>HIGH</entry>
 						</row>
+						<row>
+							<entry>
+								<b>Setuptools</b>
+							</entry>
+							<entry> </entry>
+							<entry> </entry>
+							<entry> </entry>
+							<entry> </entry>
+							<entry> </entry>
+						</row>
+						<row>
+							<entry> </entry>
+							<entry>Setuptools Detector</entry>
+							<entry>Python</entry>
+							<entry>PyPI</entry>
+							<entry>
+								<p>Files: pyproject.toml and optionally setup.cfg, or setup.py</p>
+								<p>Executables: python and pip, or python3 and pip3 specified via --detect.python.path and --detect.pip.path properties.</p>
+							</entry>
+							<entry>HIGH</entry>
+						</row>
+						<row>
+							<entry> </entry>
+							<entry>Setuptools Detector</entry>
+							<entry>Python</entry>
+							<entry>PyPI</entry>
+							<entry>
+								<p>File: pyproject.toml and optionally setup.cfg, or setup.py</p>
+								<p>Executables: python or python3 specified via --detect.python.path property.</p>
```

Comment:
The low accuracy detector just parses files and does not require either python or pip